BE/GovernmentEntities/EuropeanJurisdiction/EasternEuropeGovernmentEntitiesAndJurisdictions.rdf

```diff
@@ -1,339 +1,339 @@
+		<fibo-fnd-law-jur:hasReach rdf:resource="&lcc-3166-1;Bulgaria"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-ge-eeuj;JurisdictionOfCzechia">
+		<rdf:type rdf:resource="&fibo-fnd-law-jur;Jurisdiction"/>
+		<rdfs:label xml:lang="en">jurisdiction of Czechia</rdfs:label>
+		<rdfs:label xml:lang="cs">jurisdikce česka</rdfs:label>
+		<skos:definition>jurisdiction of the judiciary of the Czech Republic, a civil law system grounded in the Constitution of the Czech Republic, including the Constitutional Court, Supreme Court, and Supreme Administrative Court, as well as district and county courts that are divided into civil, criminal, and administrative branches</skos:definition>
+		<fibo-be-ge-ge:isJurisdictionOf rdf:resource="&fibo-be-ge-eeuj;GovernmentOfTheCzechRepublic"/>
+		<fibo-fnd-law-jur:hasReach rdf:resource="&lcc-3166-1;Czechia"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-ge-eeuj;JurisdictionOfHungary">
+		<rdf:type rdf:resource="&fibo-fnd-law-jur;Jurisdiction"/>
+		<rdfs:label xml:lang="hu">Magyarország joghatósága</rdfs:label>
+		<rdfs:label xml:lang="en">jurisdiction of Hungary</rdfs:label>
+		<skos:definition>jurisdiction of the judiciary of Hungary, a civil law system based primarily on German that is divided between courts with regular civil and criminal jurisdiction and administrative courts, including local courts, regional appellate courts, and the supreme court</skos:definition>
+		<fibo-be-ge-ge:isJurisdictionOf rdf:resource="&fibo-be-ge-eeuj;GovernmentOfHungary"/>
+		<fibo-fnd-law-jur:hasReach rdf:resource="&lcc-3166-1;Hungary"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-ge-eeuj;JurisdictionOfMoldova">
+		<rdf:type rdf:resource="&fibo-fnd-law-jur;Jurisdiction"/>
+		<rdfs:label xml:lang="en">jurisdiction of Moldova</rdfs:label>
+		<rdfs:label xml:lang="ro">jurisdicția Moldovei</rdfs:label>
+		<skos:definition>jurisdiction of the judiciary of Moldova, which is the system of courts that interprets and applies the law in Moldova, including an independent Constitutional Court</skos:definition>
+		<fibo-be-ge-ge:isJurisdictionOf rdf:resource="&fibo-be-ge-eeuj;GovernmentOfTheRepublicOfMoldova"/>
+		<fibo-fnd-law-jur:hasReach rdf:resource="&lcc-3166-1;Moldova"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-ge-eeuj;JurisdictionOfPoland">
+		<rdf:type rdf:resource="&fibo-fnd-law-jur;Jurisdiction"/>
+		<rdfs:label xml:lang="en">jurisdiction of Poland</rdfs:label>
+		<rdfs:label xml:lang="pl">jurysdykcja Polski</rdfs:label>
+		<skos:definition>jurisdiction of the judiciary of Poland, a four-tier court system composed of the Supreme Court, the Supreme Administrative Court, Common Courts (District, Regional, Appellate) and the Military Court</skos:definition>
+		<fibo-be-ge-ge:isJurisdictionOf rdf:resource="&fibo-be-ge-eeuj;GovernmentOfTheRepublicOfPoland"/>
```

Comment:
Surely this should be a reference to the Judiciary (the four tier court system) as per the definition, not the Government.

Reply:
@rivettp the issues you are relevant for the overall design of this ontology, but the current issue is just to fix the misspelled label.